src/layouts/Layout.astro

```diff
@@ -151,17 +151,15 @@ const bannerOffset = bannerOffsetByPosition[siteConfig.banner.position || 'cente
 		<div id="page-height-extend" class="hidden h-[300vh]"></div>
 	</body>
 </html>
+
 <style is:global>
 	:root {
 		--hue: var(--configHue);
 		--page-width: 75rem;
 	}
 </style>
-<style is:global define:vars={{ bannerOffset }}>
-@tailwind components;
-@tailwind utilities;
 
-@layer components {
+<style is:global define:vars={{ bannerOffset }}>
 	.enable-banner.is-home #banner-wrapper {
```

Comment:
The CSS code here needs to be wrapped inside `@layer components` to ensure `lg:is-home` is properly processed, which is needed for the banner height change on the homepage.

Reply:
https://github.com/tailwindlabs/tailwindcss/discussions/5934#discussioncomment-2793112

I have a similar problem, and `@layer components` is not working.

Reply:
I was just forgetting `@tailwind components;`.